lib/hanami/common_logger.rb

```diff
@@ -89,5 +89,21 @@ def extract_params(env)
       result.merge!(Utils::Hash.deep_stringify(env.fetch(ROUTER_PARAMS, {})))
       result
     end
+
+    # Wrapper which uses Rack's monotonic clock_time (used for began_at since Rack 2.1.0)
+    #
+    # @since 1.3.4
+    # @api private
+    class ElapsedTime
+      @clock = if Gem::Version.new(Rack::RELEASE) >= Gem::Version.new('2.1.0')
+                 -> { Rack::Utils.clock_time }
+               else
+                 -> { Time.now }
+               end.freeze
+
+      def self.call
+        @clock.call
+      end
```

Comment:
```ruby
if Gem::Version.new(Rack::RELEASE) >= Gem::Version.new('2.1.0')
  def self.call
    Rack::Utils.clock_time
  end
else
  def self.call
    Time.now
  end
end
```
what about?

Reply:
That could work as well. 👍

Reply:
Yes, that would work too :+1:  I would still stick to the first variant.